wiki/News_styling_criteria/en.md

```diff
@@ -132,6 +105,8 @@ jpeg-recompress -am smallfry <input> <output>
 
 Where `<input>` is the file name to be compressed and `<output>` is the compressed file name.
 
+Alternatively, [this site](https://ezgif.com/) can be used to convert & optimise images.
+
 #### Captions
 
 Images are not soft-captioned using Markdown except in stylistic circumstances. They may be hard-captioned where contextually appropriate.
```

Comment:
this part is irrelevant, unclear and should be removed

the part below this about alt text should be merged with the part about news post banners

Reply:
removed in 545f0d9198fbeaae803d1214ffa2ef61cbc26ea1